elastic/conf.yaml.example

```diff
@@ -29,16 +29,21 @@ instances:
   # Some managed ElasticSearch services (e.g. AWS ElasticSearch) do not expose this endpoint.
   # Set `pending_task_stats` to false if you use such a service.
 
+  # `admin_forwarder` (defaults to False) is used to signify a URL that includes a
+  # context roote needed for a forwarder application to access Elasticsearch REST services
+  # for example: https://www.ibm.com/support/knowledgecenter/SSFTN5_8.5.6/com.ibm.wbpm.main.doc/topics/tadm_fps_esearch.html
```

Comment:
Are there official ES docs for this? might be better to link there if possible. 😃

Reply:
Definitely a good idea! But I wasn't able to find anything in the ES docs themselves and the only places that seem to mention it are the forwarder applications themselves.